app/views/admin/loans/content/_table_question.html.slim

```diff
@@ -0,0 +1,89 @@
+.table-responsive.editable-tables
+
+  / TODO: Convert input to expected input for loan question
+  input type="hidden" data-container="breakeven"
+
+  .table-container
+    h4.title = t("loan.breakeven.products", count: 2)
+
+    table.table.table-condensed.table-striped.table-bordered.editable-table data-table="products"
+      thead
+        tr
+          th = t("loan.breakeven.product_name")
+          th.wide = t("loan.breakeven.description")
+          th = t("loan.breakeven.unit")
+          th.narrow = t("loan.breakeven.price")
+          th.narrow = t("loan.breakeven.cost")
+          th.narrow = t("loan.breakeven.quantity")
+          th.narrow: .actions: a data-action="add": i.fa.fa-plus-circle
+      tbody
+        / Hidden row is used to generate new rows
+        / Is at top instead of bottom to not affect striped rows
+        tr.ui-state-default.hidden data-group="product"
```

Comment:
FYI, I did this part. Sorry, Liz, it seems you were excited about it, but I thought I would be able to take care of all these changes. Alas.